test/cmd/process.sh

```diff
@@ -33,7 +33,7 @@ os::cmd::expect_failure_and_text 'oc process template-name --parameters -t somev
 os::cmd::expect_failure_and_text 'oc process template-name key=value key=value' 'provided more than once: key'
 os::cmd::expect_failure_and_text 'oc process template-name --value=key=value --value=key=value' 'provided more than once: key'
 os::cmd::expect_failure_and_text 'oc process template-name key=value --value=key=value' 'provided more than once: key'
-os::cmd::expect_failure_and_text 'oc process template-name key=value other=foo --value=key=value --value=other=baz' 'provided more than once: key, other'
+os::cmd::expect_failure_and_text 'oc process template-name key=value other=foo --value=key=value --value=other=baz' 'provided more than once: key=value, other=foo'
```

Comment:
Not sure the change in the error message makes sense here -- the _keys_ are provided more than once